daft/execution/stateful_actor.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import logging
+import multiprocessing as mp
+from typing import TYPE_CHECKING
+
+from daft.expressions import Expression, ExpressionsProjection
+from daft.table import MicroPartition
+
+if TYPE_CHECKING:
+    from multiprocessing.connection import Connection
+
+    from daft.daft import PyExpr, PyMicroPartition
+
+logger = logging.getLogger(__name__)
+
+
+def initialize_actor_pool_projection(projection: ExpressionsProjection) -> ExpressionsProjection:
```

Comment:
Isn't this code similar/shared with the PyRunner? Should we use this there as well.

Reply:
It's sorta similar but there's some differences, including not setting `CUDA_VISIBLE_DEVICES`. I could share the rest though

Reply:
Updated pyrunner to use this function